wsmaster/che-core-api-factory-github-common/src/main/java/org/eclipse/che/api/factory/server/github/AbstractGithubPersonalAccessTokenFetcher.java

```diff
@@ -140,7 +141,12 @@ public PersonalAccessToken fetchPersonalAccessToken(Subject cheSubject, String s
       Optional<Pair<Boolean, String>> valid =
           isValid(
               new PersonalAccessTokenParams(
-                  scmServerUrl, tokenName, tokenId, oAuthToken.getToken(), null));
+                  scmServerUrl,
+                  OAUTH_PROVIDER_NAME,
```

Comment:
Should we use ``this.providerName` here since we have 2 github providers: `github` & `github2`?

Reply:
I think that having `github2` as a provider name value would be confusing for users. Users would be able to distinguish the tokens by the `endpoint` value.

Reply:
Also in the `add token` form we have only one `GitHub` option, so if user manually adds a token for the second github, the `Provider` value is `github` but not `github2`.